packages/http/src/operations.ts

```diff
@@ -221,7 +221,8 @@ function getHttpOperationInternal(
 
   const httpOperation: HttpOperation = {
     path: route.path,
-    pathSegments: route.pathSegments,
+    uriTemplate: route.uriTemplate,
+    pathSegments: [], // TODO: ?
```

Comment:
TODO: were you going to add the segments or leave empty?

Reply:
ah yeah was waiting to figure out if we should just remove that prop or leave it empty. I feel like it had 0 value because it was not really the pathSegment it was more all the `@route` used to build the final one(including the parent namespace/interface recursively)

So don't feel like it had any use outside of how the http and rest library used it to resolve the final route.

Reply:
Ah gotcha, makes sense we don't really need it anymore if only http/rest used it prior to these changes.